fldigi.yml

```diff
@@ -0,0 +1,37 @@
+app-id: org.freedesktop.fldigi
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: fldigi
+finish-args:
+  - --share=ipc
+  - --socket=x11
```

Comment:
If it runs under wayland, switch this one to `fallback-x11`

Reply:
ok tks